### DISCUSSION TITLE
Cannot reach API endpoints

### DISCUSSION THREAD

**Discussion by noleksiv** [jasontaylordev/CleanArchitecture#651] [Q&A] [answered]
Angular's application tries to call an endpoint that is on the different port. Could someone help with that?
Have tried to add cors policy but without any result.
![image](https://user-images.githubusercontent.com/44829389/173197054-4abc29ab-96a6-40e1-922c-52366e0c2263.png)

**Comment by jasontaylordev** [comment]
I can help, but I need to know how you are launching the application. Please describe specific commands or actions you use to attempt to start the front end and/or back end.

**Comment by jasontaylordev** [comment, marked as answer]
I would encourage you to try:


```bash
dotnet new --install Clean.Architecture.Solution.Template
dotnet new ca-sln --output CaSlnTest
cd .\CaSlnTest\src\WebUI\
dotnet run
```

This will launch the back end. Open your browser at https://localhost:5001/.
The front end will be automatically started, and you should see the site load normally.

**Reply by noleksiv** [reply]
Thanks, that's helpful. Also, I've commented out the line with `UseAngularCliServer` to avoid "Failed to proxy the request..." error.

**Reply by jasontaylordev** [reply]
You're welcome! Just a heads up that  `UseAngularCliServer` is no longer used, you might have an older version of the template.
Suggest getting latest, lots of good improvements. 😀
